Objects/unicodeobject.c

```diff
@@ -6086,17 +6082,13 @@ _PyUnicode_DecodeUnicodeEscape(const char *s,
 
       error:
         endinpos = s-starts;
-        writer.min_length = end - s + writer.pos;
```

Comment:
Ohh, it's discussed. I know this may avoid unnecessary reallocation. But honestly I doubt how useful it could be.